src/FsAutoComplete.Core/FsAutoComplete.Core.fs

```diff
@@ -2,14 +2,21 @@ namespace FsAutoComplete
 
 open System
 open System.IO
-open Microsoft.FSharp.Compiler
 open Microsoft.FSharp.Compiler.SourceCodeServices
 open FSharpLint.Application
 
 module Response = CommandResponse
 
 module Commands =
-    let parse (serialize : obj -> string) (state : State) (checker : FSharpCompilerServiceChecker) file lines = async {
+    type Serializer = obj -> string
+
+    type Context = 
+        { Serialize: Serializer
+          State: State
+          TyRes : ParseAndCheckResults 
+          Checker : FSharpCompilerServiceChecker }
+
```

Comment:
Maintaining such formatting is a pain. If it were supported by a tool, I'd use it.